docs/_sources/content/world.rst.txt

```diff
@@ -143,8 +113,8 @@ As you can see, this is using an internal header, so handle it with care.
 Using the world
 ---------------
 
-Once you have successfully initialized a world, you can use ``REGoth::gWorld()``
-to access it. See ``src/world/GameWorld.hpp`` for the ``GameWorld``-class.
+Once you have successfully initialized a world, you can use created ``GameWorld``-component
```

Comment:
Same missing word as above

Reply:
That is actually build output, so this is automatically resolved when fixing the other one.